src/SFA.DAS.Support.Indexer.Worker/WebConfiguration.cs

```diff
@@ -2,15 +2,17 @@
 using SFA.DAS.Support.Common.Infrastucture.Settings;
 using SFA.DAS.Support.Indexer.ApplicationServices.Settings;
 using SFA.DAS.Support.Shared.SiteConnection;
+using System.Collections.Generic;
 
 namespace SFA.DAS.Support.Indexer.Worker
 {
     public class WebConfiguration : IWebConfiguration
     {
-        [JsonRequired] public SiteConnectorSettings SiteConnector { get; set; }
+        public SiteConnectorSettings SiteConnector { get; set; }
```

Comment:
can this be removed now?

Reply:
leaving it there incase this get released without MI been setup yet once released can remove it

Reply:
this branch and its config branch will only ever get merged at the same time so this isn't an option.

Reply:
removed